### ISSUE TITLE
Failure configuring AWS Autoscaling Policy with adjustment-type ExactCapacity and scaling-adjustment from 1 to 0 

### ISSUE THREAD

**Issue by hgontijo** [hashicorp/terraform#6276]
### Terraform Version
v0.6.14

### Affected Resource(s)
aws_autoscaling_policy

### Terraform Configuration Files
```hcl
...
resource "aws_autoscaling_policy" "ci_scaledown_conf" {
    name = "asg-scaledown"
    scaling_adjustment = 0
    adjustment_type = "ExactCapacity"
    cooldown = 30
    autoscaling_group_name = "${aws_autoscaling_group.ci_asg_conf.name}"
}
...
```

### Debug Output
https://gist.github.com/hgontijo/5005e78944ccffe8b87410269804680e

### Panic Output
n/a

### Expected Behavior
ASG policy scaling_adjustment to be set as "1" => "0".
This configuration can be done via AWS Console or cli: https://gist.github.com/hgontijo/14ad093e6a305b1ee58fba24fabf1164

### Actual Behavior
It fails.
### Steps to Reproduce
Configure ASG Policy hcl with:
   scaling_adjustment = 0
   adjustment_type = "ExactCapacity"

`terraform apply`

### Important Factoids
n/a

### References
n/a

**Comment by justintime** [comment]
I can confirm this bug is still present today in 0.7.1

**Comment by dennis-bsi** [comment]
I have a PR in for this if it can just get a quick review.

**Comment by catsby** [comment]
Fixed in #6276, thanks!
